.github/actions/build-backend/action.yml

```diff
@@ -30,11 +44,32 @@ runs:
       shell: bash
       run: ./gradlew package -x fatjar -Pshowtests
 
+    - name: Build Prime Router test report
+      working-directory: prime-router
+      run: ./gradlew jacocoTestReport -x fatjar
+      shell: bash
+
     - name: Tar router files
       working-directory: prime-router
       shell: bash
       run: tar -czvf prime-router-build.tar.gz build
 
+    - name: SonarCloud Scan
```

Comment:
I don't believe we want to add a sonarcloud scan to this.  It's redundant with build_hub.yml and would slow things down.